PYME/LMVis/Extras/pointSetGeneration.py

```diff
@@ -180,6 +197,16 @@ def OnGenEvents(self, event):
         pipeline.mdh['Camera.TrueEMGain'] = 1
         pipeline.mdh['Camera.CycleTime'] = 1
         pipeline.mdh['voxelsize.x'] = .110
+        # some info about the parameters
```

Comment:
to be really useful we need to preserve the parameters events were generated with so that upon saving as hdf the info is included as metadata